cargo-apk/src/apk.rs

```diff
@@ -139,6 +139,12 @@ impl<'a> ApkBuilder<'a> {
 
         let crate_path = self.cmd.manifest().parent().expect("invalid manifest path");
 
+        let is_debug_profile = match self.cmd.profile() {
+            Profile::Dev => true,
+            Profile::Release => false,
+            Profile::Custom(c) => todo!("Read `debug` field from `[profile.{}]`?", c),
```

Comment:
I'm a bit about this one - seems to effectively disallow custom profiles for now.

Reply:
@msiglreith Fair enough, perhaps I should just implement it in this PR or choose a "sensible default" - which would be?

Reply:
restricting it to `Profile::Dev` should be fine imo

Reply:
We're doing it the same as a default value for `manifest.android_manifest.application.debuggable`, fwiw.